core-services/prow/02_config/openshift/cluster-api-provider-openshift/_pluginconfig.yaml

```diff
@@ -6,9 +6,10 @@ approve:
 label:
   restricted_labels:
     openshift/cluster-api-provider-openshift:
-    - allowed_users:
+    - allowed_teams:
+      - openshift-team-cloud
+      allowed_users:
```

Comment:
@JoelSpeed , not sure if its correct to add all here as you are the only one currently.

Also removing vince for cleanup purposes.